src/api/delivery-needed/index.js

```diff
@@ -0,0 +1,60 @@
+const { findDeliveryNeededRequests } = require("~airtable/tables/requests");
+const { fields } = require("~airtable/tables/requests");
+const { fetchCoordFromCrossStreets } = require("./fetchCoordFromCrossStreets");
+
+exports.deliveryNeededRequestHandler = async (req, res) => {
+  const [requestObj, requestErr] = await findDeliveryNeededRequests();
+
+  if (!requestObj) {
+    return res
+      .status(400)
+      .send({ message: `Error fetching requests: ${requestErr}` });
+  }
+
+  const {
+    code,
+    crossStreetFirst,
+    crossStreetSecond,
+    meta,
+    neighborhoodAreaSeeMap
+  } = fields;
+
+  const requestsWithCoordsPromises = requestObj.map(async r => {
+    let metaJSON = {};
+    let slackChannelId;
+    const location = await fetchCoordFromCrossStreets(
+      `
+      ${r.fields[crossStreetFirst]},
+      ${r.fields[crossStreetSecond]},
+      Brooklyn
```

Comment:
I had to add Brooklyn here to restrict the results even more. Cross street data is too vague and sometimes returns results out of NY despite the locality setting.